test/erlcloud_s3_test_data.hrl

```diff
@@ -0,0 +1,507 @@
+%%%-------------------------------------------------------------------
+%%% @author <Kirill Starikov kstarikov@alertlogic.com>
+%%% @copyright (C) 2016, <Alert Logic>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 15. Feb 2016 22:21
+%%%-------------------------------------------------------------------
+-author("<Kirill Starikov kstarikov@alertlogic.com>").
+
+-define(S3_BUCKET_EVENT_XML_CONFIG,
+    {xmlElement,'NotificationConfiguration','NotificationConfiguration',[],
```

Comment:
This is an input parameter for `erlcloud_s3:get_bucket_notification` which is tested by eunit tests in this PR. It would make sense if we decided to test `get_bucket_attribute` as a whole.